docs/source/en/training/lora.mdx

```diff
@@ -175,6 +177,11 @@ accelerate launch train_dreambooth_lora.py \
   --push_to_hub
 ```
 
+It's also possible to additionally fine-tune the text encoder with LoRA. This, in most cases, leads
+to better results with a slight increase in the compute. To allow fine-tuning the text encoder with LoRA,
```

Comment:
Would it be possible to add more experimental details of `unet_lora`  and `text_encoder_lora` (and combined) in separate blogs (like [dreambooth](https://huggingface.co/blog/dreambooth))?

Reply:
We are leaving it to the community for now. But we will update if we find more :)